package.json

```diff
@@ -327,7 +327,7 @@
     "bip39-en": "1.1.1",
     "bitcoinjs-lib": "4.0.3",
     "bolt11": "1.2.4",
-    "coininfo": "4.4.0",
+    "coininfo": "https://github.com/LN-Zap/coininfo#fix/add-bech32-support",
```

Comment:
This should include the commit hash as a url fragment to ensure we know exactly which commit we are pulling.

For example

```
https://github.com/LN-Zap/coininfo#fix/add-bech32-support#83b93f6ce772aeb624c2a791022ff9288face0d7
```